docs/tutorials/05_torch_connector.ipynb

```diff
@@ -114,11 +114,8 @@
     "y_ = Tensor(y).reshape(len(y), 1)\n",
     "\n",
     "# Plot dataset\n",
-    "for x, y_target in zip(X, y):\n",
-    "    if y_target == 1:\n",
-    "        plt.plot(x[0], x[1], 'bo')\n",
-    "    else:\n",
-    "        plt.plot(x[0], x[1], 'go')\n",
+    "plt.plot(X[y == 1, 0], X[y == 1, 1], 'bo')\n",
+    "plt.plot(X[y != 1, 0], X[y != 1, 1], 'go')\n",
```

Comment:
I understand what you want to do here, but usually tutorials intend to be simple, while some true pythonic stuff may puzzle a reader, e.g. `X[y == 1, 0], X[y == 1, 1]`. My suggestion is either get back to the original code or add a comment for each line explaining what is written there and how it works.